src/config/environment/index.js

```diff
@@ -2,12 +2,12 @@ import Constants from 'expo-constants';
 import setEnvironmentVars from './envars';
 
 const { manifest, sessionId } = Constants;
-const { version } = manifest;
+const { nhie, version } = manifest;
```

Comment:
How is nhie being injected for mobile?

Reply:
Using the app.config expo provides.
https://docs.expo.io/workflow/configuration/#app-config